datetimerange/__init__.py

```diff
@@ -836,7 +836,7 @@ def __normalize_datetime_value(self, value, timezone):
     def from_range_text(
         cls,
         range_text: str,
-        separator: str = "-",
+        separator: str = " - ",
```

Comment:
This is not exactly equivalent to the current behavior.
How about make it possible to specify a separator as a regular expression.
This will allows for more flexible designations.